src/colvarproxy_io.cpp

```diff
@@ -209,60 +213,43 @@ std::ostream * colvarproxy_io::output_stream(std::string const &output_name,
   if (!io_available()) {
     cvm::error("Error: trying to access an output file/channel "
                "from the wrong thread.\n", COLVARS_BUG_ERROR);
-    return NULL;
+    return *output_stream_error_;
   }
 
-  std::ostream *os = get_output_stream(output_name);
-  if (os != NULL) return os;
-
-  if (!(mode & (std::ios_base::app | std::ios_base::ate))) {
-    backup_file(output_name);
+  if (output_streams_.count(output_name) > 0) {
```

Comment:
Could use output_stream_exists() ? There are a few instances of this.

Reply:
Done.

I also did the same for input streams, where the usage is very similar, even though it didn't come up in #481